pkg/spec/conflict.go

```diff
@@ -39,3 +40,42 @@ func createHeaderInConflictMsg(path *field.Path, in, in2 interface{}) string {
 func (c conflict) String() string {
 	return c.msg
 }
+
+// conflictSolver will get 2 types and returns
+// -1 - types conflict can't be resolved
```

Comment:
I think this is more readable:
```
// conflictSolver will get 2 types and returns:
//	0 - type1 and type2 are equal
//	1 - type1 should be used
//	2 - type2 should be used
//     -1 - types conflict can't be resolved
```

Reply:
for some reason gofmt is adding a space between the `2` and the `-1`

Reply:
ok he knows best :)